ROMFS/px4fmu_common/init.d/airframes/4401_ssrc_fog_x_tmotor

```diff
@@ -78,8 +78,8 @@ param set-default BAT2_CAPACITY 22000
 # Enable LL40LS in i2c
 param set-default SENS_EN_LL40LS 2
 
-# Disable internal magnetometer
-param set CAL_MAG0_PRIO 0
+# Low priority for internal magnetometer
+param set-default CAL_MAG0_PRIO 25
```

Comment:
As you commented, unless you want to disable the mag, just remove this line altogether.